src/Commands/MathCeilingCommand.ts

```diff
@@ -0,0 +1,41 @@
+import { Import } from "../Languages/Imports/Import";
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { CommandNames } from "./CommandNames";
+import { CommandMetadata } from "./Metadata/CommandMetadata";
+import { SingleParameter } from "./Metadata/Parameters/SingleParameter";
+import { NativeCallCommand } from "./NativeCallCommand";
+
+/**
+ * Rounds a number up to the nearest integer
+ */
+export class MathCeilingCommand extends NativeCallCommand {
+  /**
```

Comment:
Stylistic nitpick: these should be indented with 4 spaces, not 2. I'll file a separate issue to look into why lint rules aren't catching that...

Reply:
Updated in 4730206d82c3ecfb9b5605e5af2b6fcbe5aca8eb